src/main/java/org/example/coursesService/CourseManagement.java

```diff
@@ -0,0 +1,33 @@
+package org.example.coursesService;
+
+import java.util.Map;
+
+public class CourseManagement {
```

Comment:
It would be really good if you defined an interface that the CourseManagement would implement. That way you are adding abstraction to your management class, and later if you want to change the class that you use for Course management you would just change the implementation class that you are using